src/libtorch.cc

```diff
@@ -589,6 +592,18 @@ ModelInstanceState::ValidateTypedSequenceControl(
 TRITONSERVER_Error*
 ModelInstanceState::ValidateInputs()
 {
+  const torch::jit::Method& method = torch_model_->get_method("forward");
+  const auto& schema = method.function().getSchema();
+  const std::vector<c10::Argument>& arguments = schema.arguments();
+
+  // Currently we only support Dict[str, Tensor] as input when there is a single
```

Comment:
We support 1 dict argument, or any number of tensor arguments. You should check for both cases because we now know how to do it. That way if user does something other than the 2 cases that we support, you can give a clear error message.

Reply:
Sure, will make that improvement.

Reply:
Done. Also added a check for matching number of expected inputs between model and the config file.